vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/ScriptSteps.java

```diff
@@ -46,10 +52,14 @@ public class ScriptSteps
      *
      * @param jsFileName Value of the <i>src</i> attribute
      * @return webElement found js script
+     * @deprecated Use step:
+     * This step "Then a javascript file with the name '$jsFileName' is included in the source code" will be removed
      */
+    @Deprecated(since = "0.6.5", forRemoval = true)
     @Then("a javascript file with the name '$jsFileName' is included in the source code")
     public WebElement thenJavascriptFileWithNameIsIncludedInTheSourceCode(String jsFileName)
     {
+        LOGGER.warn(String.format(DEPRECATED_STEP_MESSAGE, "thenJavascriptFileWithNameIsIncludedInTheSourceCode"));
```

Comment:
probably in these cases it would be be enough to use `@Replacement` annotation, also please pay attention our log messages are for end-users, not technical people, thus we should never use method names.